app/components/primer/dropdown/menu_component.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Primer
+  module Dropdown
+    # DropdownMenus are lightweight context menus for housing navigation and actions.
+    # They're great for instances where you don't need the full power (and code)
+    # of the select menu.
+    class MenuComponent < Primer::Component
+      include ViewComponent::SlotableV2
+
+      SCHEME_DEFAULT = :default
+      SCHEME_MAPPINGS = {
+        SCHEME_DEFAULT => "",
+        :dark => "dropdown-menu-dark"
+      }.freeze
+
+      DIRECTION_DEFAULT = :se
+      DIRECTION_OPTIONS = [DIRECTION_DEFAULT, :sw, :w, :e, :ne, :s].freeze
+
+      renders_many :items, lambda { |divider: false, **item_arguments|
+        item_arguments[:tag] = :li
+        item_arguments[:role] = :none if divider
+        item_arguments[:classes] = class_names(
+          item_arguments[:classes],
+          "dropdown-item" => !divider,
+          "dropdown-divider" => divider
+        )
+
+        Primer::BaseComponent.new(**item_arguments)
+      }
+
+      # @example 200|With a header
+      #   <div class="position-relative mt-2">
+      #     <%= render(Primer::Dropdown::MenuComponent.new(header: "Options")) do |c|
+      #       c.item { "Item 1" }
+      #       c.item { "Item 2" }
+      #       c.item(divider: true)
+      #       c.item { "Item 3" }
+      #       c.item { "Item 4" }
+      #     end %>
+      #   </div>
+      #
```

Comment:
Do we want to have examples for a component that isn't used directly? (I could go either way)

Reply:
I think the parent doc is fine too